lib/improver/utilities/cube_manipulation.py

```diff
@@ -832,3 +833,39 @@ def enforce_coordinate_ordering(
     elif anchor == "end":
         cube.transpose(remaining_coords+coord_dims)
     return cube
+
+
+def clip_cube_data(cube, minimum_value, maximum_value):
+    """Apply np.clip to data in a resulting cube to ensure that the limits do
+    not go beyond the limits of the original input cube.
+
+    Args:
+        cube (iris.cube.Cube):
+            The cube that has been processed and contains data that is to be
+            clipped.
+        minimum_value (float):
```

Comment:
If cube data was  integer values and min/max values integer would the resulting cube still be integer? i.e. do these need to be float?

Reply:
Nope, can be ints. Changed types to "int or float".